src/index.js

```diff
@@ -98,7 +98,11 @@ async function cli () {
   // if an input file is defined, it should take precedence, otherwise, input is
   // coming from stdin and just name the file out.svg, if it hasn't been
   // specified with the '-o' option
-    output = input ? (input + '.svg') : 'out.svg'
+    if(outputFormat) {
+      output = input ? (input + '.' + outputFormat) : 'out' + '.' + outputFormat
+    } else {
+      output = input ? (input + '.svg') : 'out.svg'
```

Comment:
We might as well use [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), since this project only supports newer versions of JavaSript.

```suggestion
      output = input ? (`${input}.${outputFormat}`) : `out.${outputFormat}`
    } else {
      output = input ? (`${input}.svg`) : 'out.svg'
```